frontend/public/module/k8s/network.ts

```diff
@@ -25,48 +17,41 @@ export type ClusterNetworkFeatures = {
   [key in ClusterNetworkFeature]?: boolean;
 };
 
-/**
- * Main document depicting all the features that are supported by each supported CNI type.
- * Undefined features would require to take an ambiguous action (e.g. allow the customer
- * to set the policy Egress rule in a form, but show a warning explaining that this field is
- * not available for the Openshift SDN type)
- */
-const featuresDocument: {
-  [k in ClusterNetworkType]: ClusterNetworkFeatures;
-} = {
-  OpenShiftSDN: {
-    PolicyEgress: false,
-    PolicyPeerIPBlockExceptions: false,
-  },
-  OVNKubernetes: {
-    PolicyEgress: true,
-    PolicyPeerIPBlockExceptions: true,
-  },
-  Unknown: {},
+const getFeatureState = (data: { [key: string]: string }, key: string): boolean | undefined => {
+  // Note: config map data comes as string, not bool
```

Comment:
Is this a bug or it's because the limitation of configmap types?

Reply:
not a bug, it's the same thing that we discussed a while ago, `data` in configmap is `map[string]string`